db/seeds.rb

```diff
@@ -1,2 +1,26 @@
 # define the account, which has privileges of a superadmin
-User.create(first_name: 'First', last_name: 'Last', email: 'first.last@student.hpi.uni-potsdam.de', identity_url: 'https://openid.hpi.uni-potsdam.de/user/first.last', superadmin: true)
\ No newline at end of file
+User.create(first_name: 'First', last_name: 'Last', email: 'first.last@student.hpi.uni-potsdam.de', identity_url: 'https://openid.hpi.uni-potsdam.de/user/first.last', superadmin: true)
+
+# define users
+epic_admin = User.create(first_name: 'Admin', last_name: 'Epic', email: 'admin@epic.de')
```

Comment:
In den Seeds stehen nur Instanz-Erzeugungen, die für das Inbetriebnehmen des Projektes unabdingbar sind, zB die Erzeugung eines Superadmins.
Es gibt bereits einen [add_demo_data-Task](https://github.com/hpi-swt2/wimi-portal/blob/dev/lib/tasks/db.rake). Bitte fügt die folgenden Anweisungen dort ein.